src/Umbraco.Core/Cache/Refreshers/Implement/UserCacheRefresher.cs

```diff
@@ -2,60 +2,58 @@
 using Umbraco.Cms.Core.Models.Membership;
 using Umbraco.Cms.Core.Notifications;
 using Umbraco.Cms.Core.Persistence.Repositories;
+using Umbraco.Cms.Core.Serialization;
 
 namespace Umbraco.Cms.Core.Cache;
 
-public sealed class UserCacheRefresher : CacheRefresherBase<UserCacheRefresherNotification>
+public sealed class UserCacheRefresher : PayloadCacheRefresherBase<UserCacheRefresherNotification, UserCacheRefresher.JsonPayload>
 {
-    #region Define
-
-    public static readonly Guid UniqueId = Guid.Parse("E057AF6D-2EE6-41F4-8045-3694010F0AA6");
-
-    public UserCacheRefresher(AppCaches appCaches, IEventAggregator eventAggregator, ICacheRefresherNotificationFactory factory)
-        : base(appCaches, eventAggregator, factory)
+    public UserCacheRefresher(AppCaches appCaches, IJsonSerializer serializer, IEventAggregator eventAggregator, ICacheRefresherNotificationFactory factory)
+        : base(appCaches, serializer, eventAggregator, factory)
     {
     }
 
+    public static readonly Guid UniqueId = Guid.Parse("E057AF6D-2EE6-41F4-8045-3694010F0AA6");
+
     public override Guid RefresherUniqueId => UniqueId;
 
     public override string Name => "User Cache Refresher";
 
-    #endregion
+    public record JsonPayload
+    {
+        public int Id { get; init; }
 
-    #region Refresher
+        public Guid Key { get; init; }
+    }
```

Comment:
This creates a lot of bulk, can also be a class.

Reply:
This is the general pattern used for these JsonPayloads, see `ContentCacheRefresher` & `MemberCacheRefresher` 😁 

And IMO records are the correct type here (also the plan to change the others) as this is used purely to pass data 😊

Reply:
The compiler creates a lot of unnecessary code for records, these simple types are better suited for a regular class.